README.md

```diff
@@ -16,16 +30,32 @@ Create a project using [create-comit-app](https://github.com/comit-network/creat
 
 ## SDK Overview
 
-### Communication with cnd
+The SDK enables you to create Javascript applications on top of COMIT.
+This can for example be a trustless decentralized exchange (DEX) or a multi-currency wallet. 
 
-The SDK exports the `ComitClient` class which is a first starting point for the communication with cnd. The `ComitClient` allows you to:
+The SDK offers functionality to negotiate a trade and to execute the trade after the negotiation is finished.
 
-* `sendSwaps`: when the taker initiates a swap with the maker (after successful negotiation)
-* `getNewSwaps`: when the maker has received a swap from the taker
-* `getOngoingSwaps`: for both maker and taker during the execution of the swap (fund, redeem)
-* `getDoneSwaps`: for both maker and taker to retrieve swaps that are finished
+### Trade Negotiation vs. Execution
 
-Additionally the SDK exports more fine-grained functionality to communicate with cnd, take a look at the classes in `cnd.ts` for details. 
+The SDK integrates a simple negotiation protocol that serves as a starting point for makers and takers to create and take orders.
+
+It is recommended to take a look at the [`TakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/taker_negotiator.ts) and [`MakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/maker_negotiator.ts) classes to get started.
```

Comment:
I would remove this sentence (or describe how to use it) as I would not know what to look at in these two files _to get started_.
```suggestion
```

Reply:
please see my comments below :)